src/controllers/api/clearDialogueHistoryController.ts

```
@@ -0,0 +1,23 @@
+import { getInventory } from "@/src/services/inventoryService";
+import { getAccountIdForRequest } from "@/src/services/loginService";
+import { RequestHandler } from "express";
+
+export const clearDialogueHistoryController: RequestHandler = async (req, res) => {
+    const accountId = await getAccountIdForRequest(req);
+    const inventory = await getInventory(accountId);
+    const request = JSON.parse(String(req.body)) as IClearDialogueRequest;
+    if (inventory.DialogueHistory && inventory.DialogueHistory.Dialogues) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)